Userland/Libraries/LibArchive/Zip.h

```diff
@@ -112,8 +113,8 @@ struct [[gnu::packed]] CentralDirectoryRecord {
     u16 minimum_version;
     ZipGeneralPurposeFlags general_purpose_flags;
     ZipCompressionMethod compression_method;
-    u16 modification_time;
-    u16 modification_date;
+    AK::DOSPackedTime modification_time;
+    AK::DOSPackedDate modification_date;
```

Comment:
These should use `DOSPackedTime` without the `AK::` namespace prefix after the names are exported into the global namespace.